objects/app/apps.go

```diff
@@ -22,11 +23,13 @@ import (
 )
 
 type appsCmd struct {
-	provider provider.Provider
-	client   *fnclient.Fn
+	provider         provider.Provider
+	providerAdapter  adapter.Provider
+	client           *fnclient.Fn
```

Comment:
Same question here, do we need client or can apiClientAdapter replace it?

Reply:
Same answer.